element-connector/pom.xml

```diff
@@ -21,6 +21,11 @@
 			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.netty</groupId>
+			<artifactId>netty-all</artifactId>
+			<version>4.0.28.Final</version>
```

Comment:
4.0.28 is the minimum needed due to added support for connection pools and connection pool maps. Picking a later version sounds reasonable too.